app/src/main/java/org/exoplatform/fragment/PlatformWebViewFragment.java

```diff
@@ -392,11 +392,17 @@ public void onResponse(Call call, Response response) throws IOException {
 
     @Override
     public boolean shouldOverrideUrlLoading(WebView view, String url) {
-      if (url.contains(LOGOUT_PATH)) {
-        mListener.onUserJustBeforeSignedOut();
+      if (url != null && url.contains(mServer.getShortUrl())) {
```

Comment:
It's better to change this as follows:

```java
      // For external and short links, broadcast logout event if done
      if (url.contains(LOGOUT_PATH)) {
          mListener.onUserJustBeforeSignedOut();
      }

      if (url != null && url.contains(mServer.getShortUrl())) { // url is on the server's domain, keep loading normally
        return super.shouldOverrideUrlLoading(view, url);
      } else { // url is on an external domain, load in a different fragment
        mListener.onExternalContentRequested(url);
        return true;
      }

```

In fact the first condition is to test on logout URL where we have to broadcast event and the second is for a different logic, about url redirecting.

Reply:
Okay, 
Done